demo/dataset/criteo/fg.py

```diff
@@ -0,0 +1,109 @@
+#
+# Copyright 2022 DMetaSoul
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import sys
+import yaml
+import time
+import argparse
+import subprocess
+
+from pyspark.sql import SparkSession
+
+sys.path.append('../')
```

Comment:
Don't find a `__init__.py` file in the `common` package, does it work?

Reply:
It works. Please refer to this [link](https://stackoverflow.com/questions/37139786/is-init-py-not-required-for-packages-in-python-3-3)

Reply:
I will add an empty file in this directory.